src/client/datascience/commands/activeEditorContext.ts

```diff
@@ -166,6 +167,7 @@ export class ActiveEditorContextService implements IExtensionSingleActivationSer
                 .getOrCreateNotebook({
                     identity: interactiveWindow.notebookUri,
                     resource: interactiveWindow.owner,
+                    disableUI: true,
```

Comment:
Found we were not passing anything even when a UI was required.
I made this mandatory in some places (there are way too many files to fix, but this makes it more obivous about the intent).

When we remove all of the older classes this will get cleaned up, i found we have a large number of classes that pass this around and it was not possible to fix (assuming we that incorrectly pass `false` as the dfault value in other places) all of them.